### PR TITLE
feat: response body format

### LINKED ISSUE #7320: feat: new API design for the Admin API
### Description

One of the plans in the APISIX V3 roadmap was discussed in the mailing list: new API design for the Admin API

mail list link: https://lists.apache.org/thread/jg46m1lxn2zx3gz1r77b3v6cz6v2l4bn

Now I'm going to implement it, and this task is split into the following subtasks:

- [ ] updated some list URIs(SSL, proto )
- [ ] response body format
- [ ] supports paging to get a list of objects such as route
- [ ] supports filter when getting a list of objects such as route
- [ ] additional filter fields supported by the Route object

This issue is used to track the progress of this task, and also used to record things that require collaboration with other projects (dashboard, apisix-ingress-controller)

### PR DESCRIPTION
**Pull request by tzssangglass**
### Description

 
1. remove action field in response body, it is useless
2. make the response body format simpler for list of object 
 
Fixes #7320

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by membphis — APPROVED** [review]
LGTM

File: apisix/core/etcd.lua [hunk 1]
```diff
@@ -35,6 +37,55 @@ local is_http = ngx.config.subsystem == "http"
...
+
+    local local_conf, err = fetch_local_conf()
+    if not local_conf then
+        admin_api_version = "default"
+        log.error("failed to fetch local conf: ", err)
+        return false
+    end
+
+    local api_ver = try_read_attr(local_conf, "apisix", "admin_api_version")
+    if not api_ver or (api_ver and api_ver ~= "v3") then
```
Comment (spacewander) [inline apisix/core/etcd.lua:60]:
```suggestion
    if api_ver ~= "v3" then
```

File: t/admin/response_body_format.t [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        $block->set_value("no_error_log", "[error]\n[alert]");
+    }
+});
+
+run_tests;
+
+__DATA__
+
+=== TEST 1: use v3 admin api, no action in response body
+--- yaml_config
```
Comment (spacewander) [inline t/admin/response_body_format.t:42]: We can set yaml_config in add_block_preprocessor

File: apisix/core/etcd.lua [hunk 3]
```diff
@@ -35,6 +37,55 @@ local is_http = ngx.config.subsystem == "http"
...
+        admin_api_version = "default"
+        return false
+    end
+
+    admin_api_version = api_ver
+    return true
+end
+
+
+local function to_v3(body, action)
```
Comment (soulbird) [inline apisix/core/etcd.lua:70]: `to_v3_action ` ?

Comment (tzssangglass) [reply apisix/core/etcd.lua:70]: v3 would remove `action` in response body, `to_v3_action` looks like add `action` in response body.

File: apisix/core/etcd.lua (hunk 1 above)
Comment (tzssangglass) [reply apisix/core/etcd.lua]: update

File: t/admin/response_body_format.t (hunk 2 above)
Comment (tzssangglass) [reply t/admin/response_body_format.t]: updated

**Review by soulbird — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**